app/src/androidTest/java/de/test/antennapod/ui/MainActivityTest.java

```diff
@@ -54,6 +53,8 @@ protected void setUp() throws Exception {
         // override first launch preference
         prefs = getInstrumentation().getTargetContext().getSharedPreferences(MainActivity.PREF_NAME, Context.MODE_PRIVATE);
         prefs.edit().putBoolean(MainActivity.PREF_IS_FIRST_LAUNCH, false).commit();
+
+        solo = new Solo(getInstrumentation(), getActivity());
```

Comment:
For future reference: As soon as ``getActivity()`` is called, the activity is created. We should overwrite the shared preferences before that.